Manifests/ManagedPreferencesApplications/com.fxfactory.FxFactory.plist

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://fxfactory.com/</string>
+	<key>pfm_description</key>
+	<string>FxFactory settings</string>
+	<key>pfm_documentation_url</key>
+	<string>https://docs.google.com/spreadsheets/d/1ESX5td0y0OP3jdzZ-C2SItm-TUi-iA_bcHCBvaoCumw/edit#gid=0</string>
```

Comment:
This is the Microsoft Office preferences spreadsheet?

Reply:
Just pushed changes to the above by removing the URL completely from this key. How embarrassing! Sorry!

Reply:
Just pushed changes to the above by removing the URL completely from this key. How embarassing! Sorry!